packages/core/src/system.ts

```diff
@@ -173,11 +228,54 @@ export function createSystem<T extends ActorSystemInfo>(
       if (systemId !== undefined) {
         keyedActors.delete(systemId);
         reverseKeyedActors.delete(actorRef);
+        const event = {
+          type: `@xstate.actor.unregister`,
+          systemId: systemId as string,
+          actorRef: actorRef as T['actors'][keyof T['actors']]
+        } as const;
+        registrationObservers.forEach((listener) => {
+          listener.next?.(event);
+        });
       }
     },
     get: (systemId) => {
       return keyedActors.get(systemId) as T['actors'][any];
     },
+    subscribe: (
+      nextListenerOrObserver?:
+        | ((event: RegistrationEvent<T['actors'][keyof T['actors']]>) => void)
+        | Observer<RegistrationEvent<T['actors'][keyof T['actors']]>>,
+      errorListener?: (error: any) => void,
+      completeListener?: () => void
+    ) => {
+      const observer = toObserver(
+        nextListenerOrObserver,
+        errorListener,
+        completeListener
+      );
+
+      if (rootActor._processingStatus !== ProcessingStatus.Stopped) {
```

Comment:
Copied the implementation in `createActor`. I think it would make sense to keep this since system whose root actor is stopped won't be receiving updates, right?